cli/stubs/secure.valet.conf

```diff
@@ -25,7 +25,7 @@ server {
     }
 
     access_log off;
-    error_log VALET_HOME_PATH/Log/nginx-error.log;
+    error_log /usr/local/var/log/nginx/error.log;
```

Comment:
meh -- I don't like this change. 

My valet properly writes nginx logs to `VALET_HOME_PATH/Log/nginx-error.log`, which makes it very quick and easy to do troubleshooting, because I can quickly find that file from my home directory, without having to traverse into system directories.

I'm using Valet 2.0.8 without any of the symptoms you describe in 521 or this PR.

Reply:
Hey @drbytem thanks for your comment!
I'm using 2.0.8 as well, however, even when having `/var/root/.valet` owned by my user, I can't read `/var/root/.valet/Log/nginx-error` file (permission error). Nginx, valet, php has been reinstalled from scratch (fresh install), so I'm not sure what else I have to do to make it working w/o proposed changes... 
Do you have `.vallet` under `/var/root`?

Reply:
No, my `.valet` is in my own user directory: `~/.valet`, hence no permission issues.

Reply:
@drbyte How did you install it under your user directory? I run `valet` and it tries to install under `/var/root` because bash script does `sudo valet`. Did you do manual steps while installing 2.0.8?

Reply:
I've installed and re-installed so many times when doing writing and testing of various valet PRs I don't remember what I might have done differently: as far as I can recall it was nothing special.

Reply:
Also: never install valet while in sudo or as root.

Reply:
@drbyte running `valet` in fact does `sudo` within a shell script. That's because it fails.

Reply:
Can anybody test valet by reinstalling from scratch? Because it just doesn't work.